src/System.Drawing.Common/src/System/Drawing/GdiplusNative.Unix.cs

```diff
@@ -0,0 +1,2508 @@
+﻿using System.Drawing.Drawing2D;
+using System.Drawing.Imaging;
+using System.Drawing.Text;
+using System.Runtime.InteropServices;
+using System.Runtime.InteropServices.ComTypes;
+using System.Text;
+
+namespace System.Drawing
+{
+    internal partial class SafeNativeMethods
+    {
+        internal unsafe partial class Gdip
+        {
+            private const string LibName = "libdl";
+
+            public const int RTLD_NOW = 0x002;
+
+            [DllImport(LibName)]
+            public static extern IntPtr dlopen(string fileName, int flags);
+
+            [DllImport(LibName)]
+            public static extern IntPtr dlsym(IntPtr handle, string name);
+
+            [DllImport(LibName)]
+            public static extern int dlclose(IntPtr handle);
+
+            [DllImport(LibName)]
+            public static extern string dlerror();
```

Comment:
These P/Invokes should follow the standard conventions we've used elsewhere in the repo, e.g. these should live under src/Common/Interop/..., LibName should be in the Libraries class named appropriately, etc.